annotator/static/views/rect.js

```diff
@@ -531,6 +537,8 @@ class CreationRect extends Rect {
     }
 
     onDragEnd() {
+        var a = Date.now()
+
```

Comment:
go through this PR and remove all extra lines added; the changes should be minimalistic